src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -56,13 +57,12 @@ class ClientCredentials(object):
   functions.
   """
 
-  def __init__(self, low_credentials, intermediary_low_credentials):
+  def __init__(self, low_credentials):
     self._low_credentials = low_credentials
-    self._intermediary_low_credentials = intermediary_low_credentials
 
 
-def ssl_client_credentials(root_certificates, private_key, certificate_chain):
-  """Creates a ClientCredentials for use with an SSL-enabled Channel.
+def ssl_channel_credentials(root_certificates, private_key, certificate_chain):
```

Comment:
We would want to if the expectation is that the vast majority of the time callers would set them to `None`. If it's likely to be as-often-as-not, or eight-out-of-ten, then they should probably remain required positional arguments. There's also design pressure in the direction of required positional arguments coming from the fact that we don't really expect this function to have many call sites - one or two per application?

What is our expectation of how often callers will pass non-`None` values here?